rpcs3/main.cpp

```diff
@@ -494,30 +538,46 @@ int main(int argc, char** argv)
 	{
 		sys_log.notice("Booting application from command line: %s", args.at(0).toStdString());
 
+		auto game = sstr(args.at(0));
+
 		// Propagate command line arguments
 		std::vector<std::string> argv;
 
-		if (args.length() > 1)
+		// we skip the first entry (which should be the game)
+		for (int i = 1; i < args.length(); i++)
 		{
-			argv.emplace_back();
-
-			for (int i = 1; i < args.length(); i++)
-			{
-				const std::string arg = args[i].toStdString();
-				argv.emplace_back(arg);
-				sys_log.notice("Optional command line argument %d: %s", i, arg);
-			}
+			const std::string arg = args[i].toStdString();
+			argv.emplace_back(arg);
+			sys_log.notice("Optional command line argument %d: %s", i, arg);
 		}
 
 		// Ugly workaround
-		QTimer::singleShot(2, [config_override_path, path = sstr(QFileInfo(args.at(0)).absoluteFilePath()), argv = std::move(argv)]() mutable
+		QTimer::singleShot(2, [config_override_path, game, argv = std::move(argv)]() mutable
 		{
 			Emu.argv = std::move(argv);
 			Emu.SetForceBoot(true);
-			Emu.BootGame(path, "", true);
+			if (game.starts_with('['))
+			{
+				// we'll boot from title
+				game = game.substr(1, game.find("]")-1);
```

Comment:
formatting...
also, what happens if there is no ] ?
and why even implement it if it's not supported?
The person who ends up implementing it will most likely change it anyway